workspaces/ui/src/components/dashboards/TestingDashboard.js

```diff
@@ -47,13 +47,36 @@ async function ExampleReportTestingDashboardServiceBuilder(exampleId) {
   return new ExampleReportTestingDashboardService(orgId);
 }
 
-function TestingDashboardHome() {
-  debugger
-  const routingNavigationContext = useContext(RoutingNavigationContext);
-  const {baseUrl} = routingNavigationContext;
```

Comment:
I notice you removed this. It isn't exactly necessary at this root component, but it's best to keep paths as paths, as opposed to replacing a matched subpath with the corresponding url. Also, because we will have multiple top level components reusing this, child components should be able to link transparently throughout the common subtree. For example /orgs/:orgId/reports-by-capture and /example-reports/reports-by-capture are both base urls that we will have, but all of the links internally should be independent of that, i.e. `<Link to={`${baseUrl}/captures/capture-1`}>` which we can make a context-aware link wrapper component to simplify. So, paths should use basePath and links should use baseUrl.

Since it doesn't affect this version, no changes required. But we will most likely bring it back later.

Reply:
Makes sense! My thought was that if we need the path somewhere deeper I'd use one of ReactRouter's Hooks to get to it, since they already provide the router context, but I guess that doesn't account for the changing base url. Once we need it I'll make sure to wrap logic that in some neat way!